crates/matrix-sdk/src/test_utils/mocks.rs

```diff
@@ -317,6 +322,56 @@ impl MatrixMockServer {
         MockEndpoint { mock, server: &self.server, endpoint: RoomSendEndpoint }
     }
 
+    /// Creates a prebuilt mock for sending a state event in a room.
+    ///
+    /// Similar to: [`MatrixMockServer::mock_room_send`]
+    ///
+    /// Note: works with *any* room.
+    /// Note: works with *any* event type.
+    ///
+    /// ```
+    /// # tokio_test::block_on(async {
+    /// use matrix_sdk::{ruma::{room_id, event_id}, test_utils::mocks::MatrixMockServer};
+    /// use serde_json::json;
+    ///
+    /// let mock_server = MatrixMockServer::new().await;
+    /// let client = mock_server.client_builder().build().await;
+    ///
+    /// mock_server.mock_room_state_encryption().plain().mount().await;
+    ///
+    /// let room = mock_server
+    ///     .sync_joined_room(&client, room_id!("!room_id:localhost"))
+    ///     .await;
+    ///
+    /// let event_id = event_id!("$some_id");
+    /// mock_server
+    ///     .mock_room_send_state()
+    ///     .ok(event_id)
+    ///     .expect(1)
+    ///     .mount()
+    ///     .await;
+    ///
+    /// let responseNotMocked = room.send_raw("m.room.create", json!({ "body": "Hello world" })).await;
```

Comment:
usually we use camel_case for variable names in Rust :eyes: Can you do that here, please?